build-gradle/build.sh

```diff
@@ -254,12 +254,39 @@ gradle_build() {
   fi
 }
 
+export_built_artifacts() {
+  if ! should_deploy; then
+    return 0
+  fi
+
+  echo "=== Capturing built artifacts for attestation ==="
```

Comment:
This allows grouping and folding the related logs.
```suggestion
  echo "::group::Capturing built artifacts for attestation"
```